reckoner/course.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, course_file):
         Parse course.yml contents into instances.
         """
         self.config = Config()
-        self._dict = yaml.load(course_file)
+        self._dict = yaml.load(course_file, Loader=yaml.loader.FullLoader)
```

Comment:
I'm getting a deprecation notice in `tests/test_reckoner.py` due to not having the `Loader=` present on `yaml.load(...)`. Can we make sure the tests are updated too?